src/cryptonote_basic/cryptonote_basic.h

```diff
@@ -550,22 +550,19 @@ namespace cryptonote
     return txversion::v4_tx_types;
   }
 
-  inline enum txversion transaction_prefix::get_min_version_for_hf(uint8_t hf_version, cryptonote::network_type nettype, bool miner_tx)
+  inline enum txversion transaction_prefix::get_min_version_for_hf(uint8_t hf_version, cryptonote::network_type nettype)
   {
     nettype = validate_nettype(nettype);
     if (nettype == MAINNET) // NOTE(loki): Add an exception for mainnet as there are v2's on mainnet.
     {
       if (hf_version == cryptonote::network_version_10_bulletproofs)
         return txversion::v2_ringct;
-
-      if (miner_tx && hf_version <= cryptonote::network_version_12_checkpointing) // NOTE: Merge mining
-        return txversion::v2_ringct;
     }
 
     if (hf_version >= cryptonote::network_version_7 && hf_version <= cryptonote::network_version_9_service_nodes)
       return txversion::v2_ringct;
 
-    if (hf_version == cryptonote::network_version_10_bulletproofs)
+    if (hf_version >= cryptonote::network_version_10_bulletproofs && hf_version <= cryptonote::network_version_11_infinite_staking)
```

Comment:
Is there a reason to allow v3 txes on the v11 part of the chain?  3.0.x doesn't appear to allow these currently:

https://github.com/loki-project/loki/blob/v3.0.6/src/cryptonote_basic/cryptonote_basic.h#L550-L551

Reply:
Oh, I thought I accidentally deleted it. I found the actual problem causing the tests to fail. Updating commit